modules/questdb/src/main/java/org/testcontainers/containers/QuestDBContainer.java

```diff
@@ -0,0 +1,77 @@
+package org.testcontainers.containers;
```

Comment:
```suggestion
package org.testcontainers.questdb;
```

Going ahead, new modules should use a package name following `org.testcontainers.{module-name}`, to avoid Split Package issues with JMS.